test.disabled/ejabberd_tests/tests/vcard_SUITE.erl

```diff
@@ -617,12 +617,12 @@ prepare_vcard(ldap, JID, Fields) ->
             undefined ->
                 undefined;
             LdapField ->
-                escalus_ejabberd:rpc(eldap, mod_replace, [LdapField, [Val]])
+                escalus_ejabberd:rpc(eldap, mod_replace, [binary_to_list(LdapField), [binary_to_list(Val)]])
```

Comment:
I reverted this change basically because it required quite a lot refactoring of the whole vcard_SUITE module (there were functions pattern matching on the binary values) and there would be required additional conversions from the binary for the result stanzas anyway.... 
Generally speaking, changing this few lines of binaries to lists is not worth the effort.
If you don't agree I can struggle with that though!